Globalkmaria/rateOfReturn
# Release : 2.1.5 - Add tests, refactor, and fix purchased item input bug

…ug (#22)

* Edit : remove unused wrapping func in selectors

* Ref : Input

* Refactor : Home page

* Refactor : init state with local data

* Refactor : List page - useInitWithLocalData, useSaveChanged

* Refactor : List - delete stock and purchasedItem

* Refactor : Change file name to useDeleteStockState

* Refactor : Change file name useDeletePurchasedItemState

* Refactor : DeleteStockModal

* Refactor : Move hooks to hook folder

* Refactor : SummaryLock Component

* Format : Edit prettier options

* Add : prepare for test

* Add : Home page test

* Add : Router to test wrapper

* Refactor : Purchased Item

* Fix : Showing NaN in Summary  Ror

* Refactor : calculateGroupSummary

* Move : change test folder name to __test__

* Add : Redux test environment

* Add : Group summary test

* Add : renderWithStyle to test with onl;y style wrapper

* Fix : changeTransformValue - check max dot to be one and ignore space

* Edit : PurchasedInput to use formatted value

* Add : test for Input component

* Fix : delete stock, purchase item feature

* Add : move to Home when Logo is clicked

* Edit : change utils.test file name

* Fix : PurchasedInput - to use transform pure value

* Readme : 2.1.5 update Readme
